lib/log/RolarProgressLog.ts

```diff
@@ -123,13 +122,23 @@ export class RolarProgressLog implements ProgressLog {
                     method: HttpMethod.Post,
                     body: {
                         host: os.hostname(),
-                        content: postingLogs,
+                        content: postingLogs || [],
```

Comment:
Haven't you already confirmed that `postingLogs` is truthy and has a length greater than zero?

Reply:
Not when closed is true.